x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/form_validation/timing.test.ts

```diff
@@ -79,66 +69,68 @@ describe('<EditPolicy /> timing validation', () => {
     ['warm', 'cold', 'delete', 'frozen'].forEach((phase: string) => {
       const { name, value, error } = testConfig;
       test(`${phase}: ${name}`, async () => {
-        const { actions } = testBed;
-        await actions[phase as 'warm' | 'cold' | 'delete' | 'frozen'].enable(true);
+        await actions.togglePhase(phase as 'warm' | 'cold' | 'delete' | 'frozen');
```

Comment:
Thanks a lot for the review, @sabarasaba! That is a great catch, I'll open a separate PR for this. I started looking into this and noticed that we have some aria labels missing for frozen phase and it would have totally been caught with the correct interface.